docs/start.html

```diff
@@ -98,7 +98,7 @@ <h3 class="block-title">Step Three</h3>
                                 <h2 class="section-title">Configure</h2>
                                 <div id="connection"  class="section-block">
                                     <h3 class="block-title">Connection Details</h3>
-                                    <p>First enter the connection details for your etherlynk server as provided by your system administrator. The server is the FQDN of the server with an optional http port number. For example, myserver:7443.
```

Comment:
The name is really Etherlynk   https://github.com/Traderlynk/Etherlynk

Reply:
Oops :) Thanks for spotting that!